src/app/Dashboard/AutomatedAnalysis/AutomatedAnalysisConfigForm.tsx

```diff
@@ -65,60 +61,43 @@ import {
   TextVariants,
   ValidatedOptions,
 } from '@patternfly/react-core';
-import { CogIcon } from '@patternfly/react-icons';
 import * as React from 'react';
-import { Link } from 'react-router-dom';
+import { useTranslation } from 'react-i18next';
 import { concatMap, filter, first } from 'rxjs';
 interface AutomatedAnalysisConfigFormProps {
-  onCreate?: () => void;
-  onSave?: () => void;
   isSettingsForm: boolean;
```

Comment:
I think we can rename this to `noTitle` or something similar. Then, form embeded in the wizard can pass also pass `false`, which remove the title. How about that?

**Before**

![Screenshot from 2023-02-15 04-54-37](https://user-images.githubusercontent.com/68053619/218994436-32a4a19c-82e0-4bc8-8b69-8484fb1006d9.png)

**After**
![Screenshot from 2023-02-15 04-52-37](https://user-images.githubusercontent.com/68053619/218994475-ad729060-64cd-4516-b291-41df6e509ba7.png)